Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml

```diff
@@ -2634,9 +2759,3 @@ tests:
 fromversion: 5.0.0
 contentitemexportablefields:
   contentitemfields: {}
-system: true
```

Comment:
I think this should be re-added - seems like new platform stuff 🤷

Reply:
done